packages/office-ui-fabric-react/src/components/Link/Link.scss

```diff
@@ -54,7 +54,7 @@ button.root {
 
   @include high-contrast {
     // Using Highlight here so we still have a themeable system color that is different from both window text and link text.
```

Comment:
This should actually be Hotlight

Reply:
@betrue-final-final What is Hotlight? I don't see it on [this list](https://www.sitepoint.com/css-system-styles/) of system colors.